.gitignore

```diff
@@ -0,0 +1,18 @@
+.venv
+.idea
+
+# Ignore Python compiled files
+*.pyc
+__pycache__/
+
+# Ignore environment files
+.env
+
+# Ignore SQLite database files
+*.sqlite3
+
+# Ignore Django migration files
+**/migrations/*.py
+**/migrations/*.pyc
```

Comment:
Do not ignore migrations, they should be included in the project. Include them in the PR.